build/csslint/xfail.list

```diff
@@ -58,6 +58,7 @@ apps/homescreen/bower_components/base/base.css 4 3
 apps/homescreen/bower_components/gaia-fonts/style.css 0 1
 apps/homescreen/bower_components/gaia-theme/gaia-theme.css 5 0
 apps/homescreen/style/apps-panel.css 1 0
+apps/homescreen/style/pages-panel.css 1 0
```

Comment:
Yes, unfortunately the properties related to snap positioning are still not supported.